src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -444,21 +464,55 @@ export default defineComponent({
     this.parseVideoData()
     this.checkIfWatched()
 
-    if (this.useDeArrowTitles && !this.deArrowCache) {
+    if ((this.useDeArrowTitles || this.useDeArrowThumbnails) && !this.deArrowCache) {
       this.fetchDeArrowData()
     }
+
+    if (this.useDeArrowThumbnails && this.deArrowCache && this.deArrowCache?.thumbnail == null) {
```

Comment:
As you are already checking that `this.deArrowCache` has a truthy value earlier on, you don't need the optional chaining operator here.
```suggestion
    if (this.useDeArrowThumbnails && this.deArrowCache && this.deArrowCache.thumbnail == null) {
```